tests/conftest.py

```diff
@@ -5335,7 +5352,12 @@ def alice_columnar_table_single_batch(empty_data_context):
                                 "value": event_ts_column_data[
                                     "observed_strftime_format"
                                 ],  # Pin to event_ts column
-                                "details": {"success_ratio": 1.0},
+                                "details": {
+                                    "success_ratio": 1.0,
+                                    "candidate_strings": sorted(
+                                        DEFAULT_CANDIDATE_STRINGS
```

Comment:
Any reason for sorting here? Should we sort the list to begin with?

Reply:
Sorting makes it easier to read/scan the list for a specific value. The list is already sorted and is sorted before it is returned, but this is a safeguard in case changes are made.